CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to Rust's notion of
 
 ## [Unreleased]
 
+### Added
+- `bellman::groth16::batch::Verifier` now has a `verify_multicore` method (when
+  the `multicore` feature is enabled) which will internally use the global rayon
+  thread pool to parallelize the verification of a batch of proofs.
+
+### Changed
+- The `multicore` feature now requires the `getrandom` feature of the
+  `rand_core` crate.
```

Comment:
Hmm, does this count as a breaking change? Should we instead place this verifier behind a separate feature flag that enables both `multicore` and `getrandom`?